src/EasyAbp.Abp.TagHelperPlus/EasyAbp/Abp/TagHelperPlus/TagHelpers/TagHelperPlusAbpDynamicFormTagHelperService.cs

```diff
@@ -27,12 +28,17 @@ public class TagHelperPlusAbpDynamicFormTagHelperService : AbpDynamicFormTagHelp
 
         protected override bool IsSelectGroup(TagHelperContext context, ModelExpression model)
         {
-            return base.IsSelectGroup(context, model) || IsEasySelectorGroup(model.ModelExplorer);
+            return base.IsSelectGroup(context, model) || IsEasySelectorGroup(model.ModelExplorer) || IsNullableBoolean(model.ModelExplorer);
```

Comment:
`IsNullableBoolean` is better inside the `IsSelectGroup`.

Reply:
That's a good idea，I'll try PR for the ABP framework